docs/user-guide/experimental-api.md

```diff
@@ -0,0 +1,30 @@
+# Experimental API
+
+## What is the experimental API
+
+The exerimental API is an API group indended for experimenting with and testing
+APIs. To make this possible, unlike other API groups, there is no guarantee of
+future support, compatibility, or even existence. We do this to lower the
+development costs around making new APIs, to let experiments proceed without
+ongoing maintenance costs.
```

Comment:
Is lack of support a feature of "experimental" or of "v1alpha1".  I know that it is not how we started, but I think the best explanation of where we are and where we are going is that:
 
  - "experimental" means that api groups are currently experimental.   It is just the first non-empty-string api group.   New features may go into new api groups at any time, and they may be unstable/unsupported.  We will likely phase out "experimental" and try to put new things in the "right" api group.
  - "alpha" in the version means that the feature is not supported.  

See how I explained this in #15339